app/src/main/java/com/example/triptracker/model/repository/ImageRepository.kt

```diff
@@ -44,4 +64,23 @@ class ImageRepository {
       Response.Failure(e)
     }
   }
+
+  private fun resizeBitmap(bitmap: Bitmap, maxWidth: Int = 800, maxHeight: Int = 800): Bitmap {
```

Comment:
Good idea, as the pictures are only shown in small size this is a nice optimization